### ISSUE TITLE
How to open *.kfb files? question 

### ISSUE THREAD

**Issue by piotrpiotr1** [openseadragon/openseadragon#2135]
This is slide image formats and i want to use open sea dragon to open it.

**Comment by Aiosa** [comment]
Hello, OpenSeadragon does not support any image file formats except standard `jpg`, `png` etc.. web image file formats. For OpenSeadragon, it is irrelevant how the data you have is stored (as it should be for a front-end system). 

There are [protocols/tile sources](https://openseadragon.github.io/#examples-and-features) supported (or you can create your own tile source) that define WHAT url is sent as a request and expects a standard web-compatible image format (e.g. `jpg`) in the response.

You are looking for a back-end **web server** able to handle a certain protocol (sent from OpenSeadragon) and return expceted image tile.

I guess the fastest will be to ask for help from the developers of [this system](https://the.pathology.network/) which seems to use `Zoomify` to display `*.kbf` files. However, I strongly suspect that they might as well as convert the file to a different format before using it, since when I inspected the URL of their requests it ends like `...2F3531.svs/TileGroup2/6-18-8.jpg` [what are SVS formats?](https://www.microscopesinternational.com/support/kb/article/ngn1266.aspx).

You could for example [convert them to TIFF/SVS files](https://github.com/xjw00654/convert_kfb2svs) and then use a image server that is able to handle TIFF/SVS. IIP Image server is a popular option...

**Comment by iangilman** [comment]
Indeed... You need something on your server that'll convert kfb to something that OpenSeadragon supports, either as a dynamic thing or just once at build time. I'm not familiar with that space, but presumably there are tools like that out there!
